common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -45,6 +45,7 @@
 
     public static final String PREFS_KEY_BTC_PRECISION = "btc_precision";
     public static final String PREFS_KEY_OWN_NAME = "own_name";
+    public static final String PREFS_KEY_HIDE_BALANCE_ON_LAUNCH = "hide_balance_on_launch";
```

Comment:
from the story "If the setting for Hide Balance = ON then the balance should always be hidden when returning to the home screen (regardless of the last state)"